scripts/new_project.kts

```diff
@@ -82,12 +86,19 @@ object NewProject {
         buildProjectAndRunTests()
     }
 
+    private fun showScriptVersion() {
+        val properties = File(rootPath).loadProperties(VERSION_FILE_NAME)
+        val scriptVersion = properties.getProperty(SCRIPT_VERSION_PROPERTY_NAME) as String
+        showMessage(message = "=> Running new project script version $scriptVersion... \uD83D\uDC4B")
```

Comment:
What if we place the emoji at the beginning for parity with the other messages? 😄 
```suggestion
        showMessage(message = "=> \uD83D\uDC4B Running new project script version $scriptVersion...")
```